pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -53,6 +52,13 @@ func runDbCommand(command func(commandLine utils.CommandLine, sqlStore *sqlstore
 func runPluginCommand(command func(commandLine utils.CommandLine) error) func(context *cli.Context) error {
 	return func(context *cli.Context) error {
 		cmd := &utils.ContextCommandLine{Context: context}
+
+		cfg := setting.NewCfg()
+
+		if context.IsSet("debug") {
+			cfg.LogConfigSources()
+		}
```

Comment:
Drop these changes.

Reply:
May I ask why? This is set up similarly to the `runDbCommand` counterpart; Might not this prove useful to debug?